app/src/main/kotlin/lv/rigadevday/android/repository/model/other/Venue.kt

```diff
@@ -4,11 +4,12 @@ import com.google.firebase.database.IgnoreExtraProperties
 
 @IgnoreExtraProperties
 data class Venue(
-    val title: String = "",
-    val name: String = "",
-    val address: String = "",
-    val web: String = "",
-    val email: String = "",
-    val description: String = "",
-    val coordinates: String = ""
+        val title: String = "",
```

Comment:
No need to change this, also pretty sure that wifi field will not be there in the end. We will embed wifi password into description if necessary.